repositories/enemies/EnemyTableRepo.py

```diff
@@ -54,3 +54,16 @@ def insertMissing(cls):
 			chance = "1",
 			questLink = "N/A",
 		))
+		# DNA Guns
+		cls.get("pincermin").drops.insert(0, ItemDrop(
```

Comment:
You should use RecipeDrop here instead of ItemDrop

Reply:
Done, good catch!